spec/lib/kiev_spec.rb

```diff
@@ -62,5 +62,66 @@
           .with(data, Kiev::Config.instance.filtered_params, Kiev::Config.instance.ignored_params)
       end
     end
+
+    describe "event with predefined severity" do
+      shared_examples "with log severity" do |severity|
+        it "has log severity" do
+          initial = described_class.log_level
+          described_class.configure do |c|
+            c.log_level = ::Logger::DEBUG
```

Comment:
Honestly, didn't get the idea of temporary setting `::Logger::DEBUG`, what is the reason?

Reply:
Info is default one. For example `debug` method  in specs will produce nothing till we change `log_level `